debezium-api-contracts/README.md

```diff
@@ -0,0 +1,2 @@
+# Debezium OpenAPI Contracts
```

Comment:
We need to decide if we generate those contracts automatically for every release and check them in to a repo (main repo or a separate one?)?